prep-html-syntax/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      HTML Syntax Excercise
+    </title>
+  </head>
+  <body>
+    <h1 style="color:green;">
+      My favorite Star Wars character is Baby Yoda.
+      <img
+        src="https://helios-i.mashable.com/imagery/articles/05Ym34W6tH3CyuWYOCbpCAr/hero-image.fill.size_1248x702.v1623391769.jpg"
+        alt="Baby Yoda">
+    </h1>
+    </body>
+  </body><p style="color:red">You never what you're going to get from him. Plus, he's cute.</p>
```

Comment:
Be sure to put all visible content inside of the `<body> </body>` of the document. Also, you have an extra closing `</body>` tag.

Reply:
Hey Tim,

Thanks for the feedback. I went ahead, updated the code, and repush
![Screen Shot 2022-01-10 at 11 21 09 PM](https://user-images.githubusercontent.com/97268025/148899546-add6aecf-a846-4587-9ae4-818be014706c.png)
ed it to the GitHub. See attached the screenshot of my updated browser.